packages/compass-home/package.json

```diff
@@ -6,152 +6,66 @@
   "description": "Home",
   "main": "lib/index.js",
   "exports": {
-    "webpack": "./src/index.js",
+    "webpack": "./src/index.ts",
     "require": "./lib/index.js"
   },
   "scripts": {
-    "clean": "rimraf lib",
+    "clean": "rimraf ./lib",
     "precompile": "npm run clean",
-    "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js",
-    "prestart": "electron-rebuild --force --only keytar",
-    "start": "webpack-dev-server --config ./config/webpack.dev.config.js",
-    "start:watch": "npm run clean && webpack --config ./config/webpack.watch.config.js",
-    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
-    "test:watch": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" --watch",
-    "test:karma": "xvfb-maybe cross-env NODE_ENV=test karma start",
+    "prewebpack": "rm -rf ./lib",
+    "webpack": "webpack-compass",
+    "node-rebuild": "node ../../scripts/rebuild.js keytar",
+    "compile": "npm run webpack -- --mode production",
+    "prettier": "prettier",
+    "test:node": "mocha",
```

Comment:
IIRC so far we decided to test plugins both in node and electron environments, should we add test:electron here in addition to that?

Reply:
We aren't testing anything that would differ in an electron environment with the tests currently, but I think it makes sense at least to future proof ourselves for those cases. Added an electron env, it's erroring now but looking into.